### ISSUE TITLE
Mainstream packaging per OS

### ISSUE THREAD

**Issue by cloud-8** [Codaone/DEXBot#463]
For DEXbot to really reach a wider audience we need to do away with the need to install anything via cmd line for the GUI. We also always face problems related to necessary packages not being present or installed on people's computers which results in all manner of errors. 

For windows the common install mechanism is via an install wizard, this should be the case. 
For OSX it is a .dmg file that is dragged into the 'applications' folder. 
For Linux the most common distro is ubuntu based therefore perhaps a .deb file is in order or to host it on the Ubuntu app store. 

This would help more user get into using dexbot and ultimate add more liquidity to the Bitshares DEX. Most people fall at the first hurdle, that of installing it. This can help resolve that.

**Comment by MarkoPaasila** [comment]
No, let's not "do away with" cli installation, but possibly gradually add/improve builds and installers for the most common operating systems.

**Comment by thehapax** [comment]
cli needs to stay.

**Comment by cloud-8** [comment]
Do away with the need. Not do away with the cmd at all. Lets actually make a working product for the masses please...

**Comment by cloud-8** [comment]
This has been with the 'in discussion' label for 2 weeks and also has the 'huge task' label. Yet I dont see how this is more complicated than the existing discussions on implementing more obscure strategies. Firstly lets have a working product for everyone then lets have a additional strategies, its about prioritisation.

**Comment by PermieBTS** [comment]
This is an important issue and the install packages will be developed. Not just now, but once we've ironed out the kinks in the software and tackled the higher-priority testing process it will be done.
Better to fine tune the product before "launching" for mainstream via install packages

**Comment by PermieBTS** [comment]
Remove label: Requires Discussion
